grafana/provisioning/dashboards/metrics.json

```diff
@@ -964,6 +1013,24 @@
                 "skipUrlSync": false,
                 "sort": 0,
                 "type": "query"
+            },
+            {
+                "current": {
+                    "selected": false,
+                    "text": "Loki",
+                    "value": "P8E80F9AEF21F6940"
+                },
+                "hide": 0,
+                "includeAll": false,
+                "label": "Logs Data Source",
+                "multi": false,
+                "name": "logs_ds",
+                "options": [],
+                "query": "loki",
+                "refresh": 1,
+                "regex": "",
+                "skipUrlSync": false,
+                "type": "datasource"
```

Comment:
I think it might be better looking to hide the variable (but it's just me).

Reply:
I decided to make it that way to be similar to Prometheus data source.

Reply:
I refered to both loki and prometheus variables (at least I've never seen one on dashboard to be able to pick one) but you don't have to do it. I'd rearrange the order though - put prometheus' and loki's variables next to each other.